config/default.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
   theme: 'dark',
   currency: 'USD',
   locale: 'en',
+  timeDisplayMode: '12Hour',
```

Comment:
Can you add comment for this line that lists out the supported values please?

Reply:
Done @mrfelton